algosdk/dryrun_results.py

```diff
@@ -0,0 +1,85 @@
+from typing import List
+
+
+class DryrunResponse:
+    def __init__(self, drrjson: dict):
+        # These are all required fields
+        self.error = drrjson["error"]
+        self.protocol = drrjson["protocol-version"]
+        self.txns = [DryrunTransactionResult(txn) for txn in drrjson["txns"]]
+
+
+class DryrunTransactionResult:
+    def __init__(self, dr):
+        # Only required field
+        self.disassembly = dr["disassembly"]
+
+        # These are all optional
```

Comment:
Nit. with so many keys, it's probably easier to read and maintain if you do something like:
```python
fields =["disassembly", "app-call-messages", .... ]
for field in fields:
    if field in dr:
        setattr(self, field, dr[field]) 
```
then later, you can re-set the special fields `app-call-trace` and `logic-sig-trace`